bin/crew

```diff
@@ -496,89 +469,61 @@ def cache_build
   end
 end
 
-def const(var)
+def const(var = nil)
   if var
-    value = eval(var)
+    value = Object.const_get(var.to_sym)
     puts "#{var}=#{value}"
   else
-    @ruby_default_constants = %w[
-      ARGF
-      ARGV
-      CROSS_COMPILING
-      DOC
-      ENV
-      GC
-      IO
-      JSON
-      OpenSSL
-      Q
-      R
-      RUBY_COPYRIGHT
-      RUBY_DESCRIPTION
-      RUBY_ENGINE
-      RUBY_ENGINE_VERSION
-      RUBYGEMS_ACTIVATION_MONITOR
-      RUBY_PATCHLEVEL
-      RUBY_PLATFORM
-      RUBY_RELEASE_DATE
-      RUBY_REVISION
-      RUBY_VERSION
-      RubyVM
-      S
-      STDERR
-      STDIN
-      STDOUT
-      StringIO
-      TOPLEVEL_BINDING
-      URI
-    ]
-    # Get all constants
-    @constants = Module.constants.grep(/[[:upper:]]$/)
-    # Reject all constants which match the default list
-    @constants = @constants.map(&:to_s).reject { |e| @ruby_default_constants.any? { |f| /\A#{e}\z/ =~ f } }
+    # Get all constants defined by const.rb
```

Comment:
Do we want to be able to override the built-in ruby constants?

(Obviously this does simplify things ...)

Reply:
Umm, not sure what you mean

The original logic here is to get all constants defined by `const.rb` by rejecting all default constants (this expecting the remaining constants are all crew constants)

Reply:
Oh, do you mean unsetting and redefining the default constants in `const.rb`?

The new logic can still handle it :) (since redefining will change the source location of that constant to `const.rb`) (the new logic works by checking where was the constant defined)